<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LPConfig</class>
 <widget class="QDialog" name="LPConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Dataset</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Data Set:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_dataset">
       <property name="text">
        <string notr="true">sample</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_local">
      <attribute name="title">
       <string>Local Snapshots</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Schedule:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="combo_local_schedule">
           <item>
            <property name="text">
             <string>Automatic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Daily</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hourly</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>30 minutes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10 minutes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5 minutes</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QTimeEdit" name="time_local_daily">
           <property name="maximumTime">
            <time>
             <hour>23</hour>
             <minute>0</minute>
             <second>0</second>
            </time>
           </property>
           <property name="minimumTime">
            <time>
             <hour>1</hour>
             <minute>0</minute>
             <second>0</second>
            </time>
           </property>
           <property name="displayFormat">
            <string>@ h AP</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QSpinBox" name="spin_local_numkeep">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>800</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_local_keepunits">
           <item>
            <property name="text">
             <string>Days</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Total</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_local_keep">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Keep:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_remote">
      <attribute name="title">
       <string>Replication</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupReplicate">
         <property name="title">
          <string>Replicate on a Remote System</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="push_scanNetwork">
              <property name="text">
               <string>&amp;Scan Network</string>
              </property>
              <property name="icon">
               <iconset resource="lPreserve.qrc">
                <normaloff>:/images/network-wired.png</normaloff>:/images/network-wired.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Host Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineHostName"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>User Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineUserName"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>SSH Port</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QSpinBox" name="spinPort">
              <property name="maximum">
               <number>999999</number>
              </property>
              <property name="value">
               <number>22</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Remote Dataset</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineRemoteDataset"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Frequency</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QComboBox" name="combo_remote_schedule">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="sizeAdjustPolicy">
                  <enum>QComboBox::AdjustToContents</enum>
                 </property>
                 <item>
                  <property name="text">
                   <string>Sync with snapshots</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Daily</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Hourly</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>30 Minutes</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>10 Minutes</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Manual</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QTimeEdit" name="time_replicate">
                 <property name="displayFormat">
                  <string>@ h AP</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Preferred</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="tool_apply">
       <property name="text">
        <string>&amp;Apply</string>
       </property>
       <property name="icon">
        <iconset resource="lPreserve.qrc">
         <normaloff>:/images/backup-ok.png</normaloff>:/images/backup-ok.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tool_cancel">
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="icon">
        <iconset resource="lPreserve.qrc">
         <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="lPreserve.qrc"/>
 </resources>
 <connections/>
</ui>
